packages/spec/src/index.ts

```diff
@@ -93,7 +101,7 @@ export declare namespace v1 {
   /**
    * The base types interface of Standard Schema.
    */
-  interface StandardTypes<Input, Output> {
+  export interface Declaration<Input = unknown, Output = Input> {
```

Comment:
I am careful about adding defaults to generics to prevent users from forgetting to add them. Is there a use case where defaults for this type are beneficial?

Reply:
They're definitely beneficial if someone wants to use it as a standalone type.

Another option is to create a type like `Declaration.any` or similar that is essentially the type with default values passed.

If this is a type we ever expect external users to reference, I think adding them as a convenience is nice. If it's just internal, perhaps it shouldn't be exported at all and we can remove the defaults.

Reply:
When not needed, I prefer to keep the API interface minimal and simple. I would probably not add any defaults and only change it if people complain, but in the end it is not a big deal to me. Either way is fine.

What do you think... should we only export types we expect people to use or should we export everything by default?

Reply:
I'm happy with `unknown` generics here

Reply:
OK, let's leave it the way it is then. 👍